examples/FirebaseNeoPixel_ESP8266/README.md

```diff
@@ -54,7 +54,7 @@ firebase deploy
 
 ## Play
 
-1. Go to the firebase hosting URL: `firebase-name.firebaseapp.com`.
+1. Go to the firebase hosting URL: `yourproject.firebaseapp.com`.
```

Comment:
actually I was suggesting to rename the firebaseio.com url to match what's in the .ino file, but updating both also sounds good!